test/runtests.jl

```diff
@@ -710,7 +724,7 @@ end
     x = Any[1, 2, 4, 10]
     y = Any[1, 2, 4, 10//1]
     @test var(x) === 16.25
-    @test var(y) === 65//4
+    @test var(y) == 65//4
```

Comment:
```suggestion
    @test var(y) === 16.25
```
Technically this qualifies as a breaking change I guess.

Reply:
True, however it is not obvious to me whether one or the other is more "right"

Reply:
Yes, the point is that I'm not sure whether our policy allows changing things like this in a minor release.

Reply:
Note that this change restores consistency
```
julia> var(y,dims=1)[];var(y,dims=1)[]===16.25
true
```
So maybe calling it "breaking" is too strong.